README.md

```diff
@@ -70,35 +70,23 @@ for details.
 ## Configuring a external NFS mount to write files to external NFS mount.
```

Comment:
Replace line 70 with this text:

## Writing to an NFS volume

Use this procedure to configure the Flink Stream to File job to write to any Kubernetes Persistent Volume, such as a remote NFS volume.

Reply:
Changed